src/agent/src/rpc.rs

```diff
@@ -537,6 +554,54 @@ impl AgentService {
             }
         }
     }
+
+    // When being passed an image name through a container annotation, merge its
+    // corresponding bundle OCI specification into the passed container creation one.
+    async fn merge_bundle_oci(&self, container_oci: &mut oci::Spec) -> Result<()> {
+        if let Some(image_name) = container_oci
+            .annotations
+            .get(&ANNO_K8S_IMAGE_NAME.to_string())
+        {
+            if let Some(container_id) = self.sandbox.clone().lock().await.images.get(image_name) {
+                let mut image_oci_config_path = PathBuf::from(CONTAINER_BASE);
+                image_oci_config_path.push(&container_id);
+                image_oci_config_path.push(&String::from("config.json"));
```

Comment:
Nit: This well-known filename string should really be a `const` (as it's also used in another place in this file).

Reply:
Fixed, thanks.